src/main/java/org/traccar/protocol/LaipacProtocolDecoder.java

```diff
@@ -110,17 +110,26 @@ private String decodeAlarm(String event) {
 
     private String decodeEvent(String event, Position position) {
 
-        if (event.length() == 1) {
-            char inputStatus = event.charAt(0);
-            if (inputStatus >= 'A' && inputStatus <= 'D') {
-                int inputStatusInt = inputStatus - 'A';
-                position.set(Position.PREFIX_IN + 1, inputStatusInt & 1);
-                position.set(Position.PREFIX_IN + 2, inputStatusInt & 2);
-                return null;
-            }
+        if (event.length() != 1) {
```

Comment:
Please keep the same structure as before. That way it's easier to see the difference.